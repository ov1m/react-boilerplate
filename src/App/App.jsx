import React from 'react';

const App = () => <div>render react 2</div>;

export default App;
