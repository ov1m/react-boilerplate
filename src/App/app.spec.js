import React from 'react';
import App from './App';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
