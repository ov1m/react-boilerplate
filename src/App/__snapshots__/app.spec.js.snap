// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  render react 2
</div>
`;
